src/site/content/en/blog/largest-contentful-paint/index.md

```diff
@@ -0,0 +1,264 @@
+
+### How are element layout and size changes handled?
+
+To keep the performance overhead of calculating and dispatching new performance
+entries low, changes to an element's size or position do not generate new LCP
+candidates. Only the element's initial size and position in the viewport is
+considered.
+
+This means images that are initially rendered off-screen and then transition
+on-screen may not be reported. It also means elements initially rendered in the
+viewport that then get pushed down, out of view will still report their
+initial, in-viewport size.
+
+An element can only be removed from LCP consideration if it's removed from the
+DOM.
+
+### Examples
+
+Here are some examples of when the Largest Contentful Paint occurs on a few
+popular websites:
+
+![Largest Contentful Paint timeline from cnn.com](lcp-cnn-filmstrip.png)
+
+![Largest Contentful Paint timeline from techcrunch.com](lcp-techcrunch-filmstrip.png)
+
+In both of the timelines above, the largest element changes as content loads.
+In the first example, new content is added to the DOM and that changes what
+element is the largest. In the second example, the layout changes and content
+that was previously the largest is removed from the viewport.
+
+While it's often the case that late-loading content is larger than content
+already on the page, that's not necessarily the case. The next two examples
+show the Largest Contentful Paint occurring before the page fully loads.
+
+![Largest Contentful Paint timeline from instagram.com](lcp-instagram-filmstrip.png)
+
+![Largest Contentful Paint timeline from google.com](lcp-google-filmstrip.png)
+
+In the first example, the Instagram logo is loaded relatively early and it
+remains the largest element even as other content is progressively shown. In
+the Google search results page example, the largest element is a paragraph of
+text that is displayed before any of the images or logo finish loading. Since
+all the individual images are smaller than this paragraph, it remains the
+largest element throughout the load process.
+
+{% Aside %}
+  In the first frame of the Instagram timeline, you may notice the camera logo
+  does not have a green box around it. That's because it's an `<svg>` element,
+  and `<svg>` elements are not currently considered LCP candidates. The first
+  LCP candidate is the text in the second frame.
+{% endAside %}
+
+## How to measure Largest Contentful Paint in JavaScript
+
+The following code block shows how to measure LCP in JavaScript and report it
+to an analytics service when the user leaves the tab:
+
+```js
+// Create a variable to hold the latest LCP value (since it can change).
+let lcp;
+
+// Create the PerformanceObserver instance.
+const po = new PerformanceObserver((entryList) => {
+  const entries = entryList.getEntries();
+  const lastEntry = entries[entries.length - 1];
+
+  // Update `lcp` to the latest value, using `renderTime` if it's available,
```

Comment:
Remove the backticks from the code comments. They don't do anything.

Reply:
I think they add meaning. They emphasize that I'm referring to variables in the example.

Reply:
If that's what you were going for, then they need to be quotation marks because that's a more standard convention. As it is, someone might mistake this for us just being sloppy.

Reply:
Lots of doc generators parse comments as markdown. I see the usage of markdown all the time in code comments, so I don't think anyone would find this weird.